src/main/java/com/mixpanel/android/mpmetrics/ResourceReader.java

```diff
@@ -109,6 +125,16 @@ private static void readClassIds(Class<?> platformIdClass, String namespace, Map
         }
     }
 
+    protected List<String> getLocalClassNames() {
```

Comment:
this defeats the purpose of having subclasses. definitely not recommended.

Reply:
I misunderstood what you initially meant. I was pretty confused as to why we would need subclasses but now I see I just horribly misinterpreted what you meant. Fixing now